badge_server/README.rst

```diff
@@ -1,8 +1,6 @@
 Badge Server
 ============
 
-.. image:: http://35.226.8.89/one_badge_image?package=tensorflow
-
```

Comment:
Is this supposed to be replaced with the new endpoint?

Reply:
I felt like this was added accidentally from previous PR. So removing it...